recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - astroid >=2.7
```

Comment:
I think this needs to be bumped to `astroid >= 3`

Upstream is:

```
    'astroid>=2.7;python_version<"3.12"',
    'astroid>=3;python_version>="3.12"',
```
astroid > 3 is on conda-forge for older python's so we should be good to go.